frontend/openchat-client/src/openchat.ts

```diff
@@ -2645,6 +2656,14 @@ export class OpenChat extends EventTarget {
         }
     }
 
+    freezeGroup(chatId: string, reason: string | undefined): Promise<FreezeGroupResponse> {
```

Comment:
We have no error handling on this. I think we should follow the pattern used elsewhere where we parse the response inside the client and log the specific error if anything goes wrong. Then we actually just return `Promise<boolean>` from these functions indicating binary success or failure. Then the UI will toast an error message if it gets a `false` response.